homeassistant/components/frontend/__init__.py

```diff
@@ -543,6 +544,12 @@ async def reload_themes(_: ServiceCall) -> None:
         """Reload themes."""
         config = await async_hass_config_yaml(hass)
         new_themes = config.get(DOMAIN, {}).get(CONF_THEMES, {})
+
+        try:
+            THEME_SCHEMA(new_themes)
+        except vol.Invalid as err:
+            raise HomeAssistantError(f"Failed to reload themes: {err}") from err
```

Comment:
We could add translations here in the future

Reply:
I thought about it but I don't know if its weird that the message would then be in two languages, as the vol error will be an english string?